lib/pengine/native.c

```diff
@@ -655,7 +657,10 @@ pcmk__native_output_string(pe_resource_t *rsc, const char *name, pe_node_t *node
         const char *desc = crm_element_value(rsc->xml, XML_ATTR_DESC);
 
         if (desc) {
-            pcmk__add_word(&outstr, 0, desc);
+            g_string_append(outstr, "(");
```

Comment:
need a space before (pcmk__add_word() adds one automatically)

alternatively you can use g_string_append_printf() with " (%s)"